index.js

```diff
@@ -84,31 +84,32 @@ function aliases(aliases) {
 }
 
 function defaults(defaults, aliases) {
+  var out = {}
```

Comment:
@jridgewell I think this will have a bit of a perf hit. 😅

Reply:
I didn't test, but I'd imagine it'll be a very tiny hit, if one at all.

Reply:
I just did, I'll push an update to the bench in a sec.